### PR TITLE
UCS/ARBITER: Fix docs

### PR DESCRIPTION
**Pull request by brminich**
## What
Minor fixes in the arbiter documentation

### CODE REVIEW

File: src/ucs/datastruct/arbiter.h [hunk 1]
```diff
@@ -269,7 +269,7 @@ static inline void ucs_arbiter_group_desched(ucs_arbiter_t *arbiter,
 }
 
 /**
- * @return whether the group does not have any queued elements.
+ * @return whether the element is queued in some of the arbiter groups.
```
Comment (yosefe) [inline src/ucs/datastruct/arbiter.h]: whether the element is queued in an arbiter group.
(an element can't be queued more than once)

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8843/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8845/ for details (Mellanox internal link).

**Comment by brminich** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8876/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8879/ for details (Mellanox internal link).

**Pull request merged**